state/cleanup.go

```diff
@@ -384,17 +384,24 @@ func (st *State) cleanupMachinesForDyingModel(cleanupArgs []bson.Raw) (err error
 		return errors.Trace(err)
 	}
 	force := args.Force != nil && *args.Force
+
+	// There are some cases below, where we should continue on destroying
+	// machines if a failure is encountered with one machine.
+	//   1) If destroy machine fails due to existing containers on it.
+	//   2) If force is in use.
+	var continueError bool
 	for _, m := range machines {
 		if m.IsManager() {
 			continue
 		}
-		if _, isContainer := m.ParentId(); isContainer {
-			continue
-		}
 		manual, err := m.IsManual()
 		if err != nil {
-			// TODO (force 2019-4-24) we should not break out here but continue with other machines.
-			return errors.Trace(err)
+			if !force {
```

Comment:
force is only supposed to be used for cloud related api errors like destroy instance failing due to the instance being hung or storage being stuck or whatever. The possible err here is due to a problem reading from mongo which is not something force is for.

Reply:
reverting this piece back

Reply:
Where is it documented that for is only for cloud related errors?  I've been unable to find that be of data in the code.